typespec/Applications.Core/containers.tsp

```diff
@@ -322,6 +322,9 @@ enum VolumePermission {
 enum IAMKind {
   @doc("Azure Active Directory")
   azure,
+
+  // Extensible
```

Comment:
Could we add a comment here that this is the way the TypeSpec team would like us to do it? In case anyone works on this in the future, they know why it's here/not to delete it

Reply:
Is there a tracking issue for this in the typespec repo?